app/views/folders/index.html.erb

```diff
@@ -5,12 +5,22 @@
   <%= link_to "New Folder", new_folder_path, class: "btn btn-primary" %>
   </p>
 <% end %>
+<%= link_to "All Folders", folders_path, class: 'btn btn-primary'%>
+<br/>
+<br/>
+<%= form_tag folders_path, :method => 'get', :class => 'form-inline' do %>
```

Comment:
Minor and purely style, but if I recall correctly I think the "hashrocket" has fallen out of fashion, with a reasonably complete replacement as of Ruby 2.2 with the JSON style hash notation - e.g. `method: 'get', class: 'form-inline'`, although I don't know if there are still places you need the hashrocket...

Reply:
Yup, generally for new stuff use the updated hash notation unless something breaks :)

Running rubocop is a great way to pick on these things too when you're working on code.  I'm try to do a better job of running it against specific files when I'm working on them.

Reply:
@markottaviani and @tobymurray Will update :)

Reply:
hash rocket for non symbols basically